### PR TITLE
sophie's code

### PR DESCRIPTION
**Pull request by steve-messing**
please review my code!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Person A should modify content here.
 
 ## Person B:
 
-Person B should modify content here.
+I am Sophie, otherwise known as Person B. This is my code.
+:)
```
Comment (agesak) [inline README.md:12]: wow love the smiley face so creative

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Person A should modify content here.
 
 ## Person B:
 
-Person B should modify content here.
+I am Sophie, otherwise known as Person B. This is my code.
```
Comment (agesak) [inline README.md:11]: I am starting a review!

File: README.md [hunk 3]
```diff
@@ -8,7 +8,8 @@ Person A should modify content here.
...
+:)
 
 ## Person C:
```
Comment (agesak) [inline README.md:14]: this is another comment!!

**Review by agesak — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (steve-messing) [reply README.md:11]: Thanks for your review! Not making any changes.

**Review by agesak — APPROVED** [review]

**Pull request merged**